README.md

```diff
@@ -1,50 +1,70 @@
-# Prism
+[![Prism is a set of packages for API mocking with OpenAPI v2 (formerly known as Swagger) and OpenAPI v3.](./examples/readme-header.svg)](https://stoplight.io/api-mocking?utm_source=github&utm_medium=prism&utm_campaign=readme)
```

Comment:
I really do not see the value of putting an image. I still vote to keep it in the way it is.

Reply:
Moreover, in this way the text is hidden from the image, in case the download succeeds — not too sure this is what we want.

Reply:
I have updated to add a description followed by the image. That way even if the image doesn't load we'll have a description.

The image is for aesthetic purposes and makes for a good reading for normal use cases i.e. laptop and phone